docs/resources.html

```diff
@@ -37,59 +40,62 @@ <h2 id="general">General</h2>
-				<p>A good case can be made that social science disciplines are characterized by different kinds of systemic inequality. For example, elitism is one source of inequality -- <a href="https://doi.org/10.1111/aman.13158" target="_blank">Kawa et al. (2019)</a> found that the top 5 or so anthropology departments in the USA all hire each others' graduates.</p>
-				<p>Some fields have developed local  resources that may have the potential to be adapted to other fields. For example, <a href="https://doi.org/10.1177/1745691620927709" target="_blank">Roberts et al. (2020)</a> examine racial inequality in psychological research to date and offer recommendations for editors and authors for working towards research that benefits from diversity in editing, writing, and participation. <a href="https://doi.org/10.31234/osf.io/6nk4x" target="_blank">Buchanan et al. (2020)</a> also discusses strategies for upending racism in psychological science. <a href="https://doi.org/10.31234/osf.io/zp9em" target="_blank">Buchanan et al. (2021)</a> proposes a Diversity Accountability Index for Journals (DAI-J) in order to increase awareness and establish accountability across psychology journals.</p>
+				<p>An argument can be made that social science disciplines are characterized by different kinds of systemic inequality. For example, elitism is one source of inequality -- <a href="https://doi.org/10.1111/aman.13158" target="_blank">Kawa et al. (2019)</a> found that the top 5 or so anthropology departments in the USA all hire each others' graduates. Another example is authors’ limited geographical distribution. Of the authors who published articles in the top five developmental psychology journals between 2006 and 2010, fewer than 3% were from countries in Central or South America, Africa, Asia, or the Middle East (<a href="https://doi-org.libezproxy2.syr.edu/10.1016/j.jecp.2017.04.017" target="_blank">Nielsen et al., 2017</a>). This limitation also extends to journal editors (<a href="http://www.doi.org/10.31235/osf.io/4nq97" target="_blank" >Altman & Cohen, 2021</a>).</p>
+				<p>Some fields have developed local resources that may have the potential to be adapted to other fields. For example, <a href="https://doi.org/10.1177/1745691620927709" target="_blank">Roberts et al. (2020)</a> examine racial inequality in psychological research to date and offer recommendations for editors and authors for working towards research that benefits from diversity in editing, writing, and participation. <a href="https://doi.org/10.31234/osf.io/6nk4x" target="_blank">Buchanan et al. (2020)</a> also discusses strategies for upending racism in psychological science. <a href="https://doi.org/10.31234/osf.io/zp9em" target="_blank">Buchanan et al. (2021)</a> proposes a Diversity Accountability Index for Journals (DAI-J) in order to increase awareness and establish accountability across psychology journals.</p>
```

Comment:
```suggestion
				<p>Some fields have developed local resources that could be adapted to other fields. For example, <a href="https://doi.org/10.1177/1745691620927709" target="_blank">Roberts et al. (2020)</a> examine racial inequality in psychological research to date and offer recommendations for editors and authors for working towards research that benefits from diversity in editing, writing, and participation. <a href="https://doi.org/10.31234/osf.io/6nk4x" target="_blank">Buchanan et al. (2020)</a> also discuss strategies for upending racism in psychological science. In a different paper, <a href="https://doi.org/10.31234/osf.io/zp9em" target="_blank">Buchanan et al. (2021)</a> propose a Diversity Accountability Index for Journals (DAI-J) in order to increase awareness and establish accountability across psychology journals.</p>
```

See if these make sense